lib/teamcityBrowser.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+/**
+ * Teamcity doc reference https://confluence.jetbrains.com/display/TCD10/Build+Script+Interaction+with+TeamCity
+ *
+ * Module dependencies.
+ */
+var Base = window.Mocha.reporters.Base;
+var log = console.log;
```

Comment:
It would be great if we could have opportunity to use our log function.

The thing is, we run tests remotely and send logs to our server. So we need to send teamcity-logs too.

Let's think about it.
Maybe you could put in global scope function replacing variable `log`.
For example:
```js
window.mochaTeamcityReporter = {
    changeLog: function(newLogFunc) {
        log = newLogFunc;
    }
}
```

Or maybe some another solution...